spec/integration/kibana/kb_entry/spec.yml

```diff
@@ -0,0 +1,27 @@
+spec:
+  additionalContents: false
+  totalContentsLimit: 50
+  contents:
+    - description: Folder containing a single knowledge base entry definition
+      type: folder
+      pattern: '^[a-z0-9_]+[a-z0-9]$'
+      required: true
+      additionalContents: false
+      contents:
+        - description: A knowledge base entry's manifest file
+          type: file
+          contentMediaType: "application/x-yaml"
+          sizeLimit: 2MB
+          name: "manifest.yml"
+          required: true
+          $ref: "./manifest.spec.yml"
+        - description: A knowledge base entry's index mapping
+          type: file
+          contentMediaType: "application/json"
+          sizeLimit: 5MB
+          name: "index-mapping.json"
+          required: true
```

Comment:
I decided to KISS and keep the index's mapping as a plain `json` file, first because I don't think adding validation on the index's mapping format or having it as a yaml file would bring much value (AFAIK we're not doing it for index templates, mappings are just defined as `object` type with allowed extra props), and I also did not use the `fields.yml` feature because we don't need, or even want, any kind of discoverability on this type of content. 

Please tell me if you think this doesn't make sense.

Reply:
So an other part of the reasoning was that each knowledge base entry can, technically, have different mappings. I'm not sure how that would play with that `fields.yml` feature, given that from my understanding, this is only a top level section in the package?

Or should we use the same format but with a different pattern, like having one fields.yml file per knowledge base entry folder (that would then replace the `index-mappings.json` file)?

Reply:
We reuse the fields definitions on different places. The more direct way to add it here would be to have a `fields` directory here, something like this:
```
        - description: Folder containing field definitions to be used as the mappings for the index template
          type: folder
          name: fields
          required: false
          $ref: "../../data_stream/fields/spec.yml"
```